py/desispec/resolution.py

```diff
@@ -94,4 +105,32 @@ def to_fits_array(self):
         """
         return self.data
 
-#- (Unit tests moved to desispec.test.test_resolution)
\ No newline at end of file
+def _gauss_pix(x, mean=0.0, sigma=1.0):
+    """
+    Utility function to integrate Gaussian density within pixels
+
+    Args:
+        x (1D array): pixel centers
+        mean (float): mean of Gaussian
+        sigma (float): sigma of Gaussian
```

Comment:
This would take more than just passing in an array for sigma, since the bin rescaling line `x = np.asarray(x, dtype=float) / sigma` would then need to become a matrix operation with other matrix operations downstream.  This is only worth it if the loop is a slow limiting factor.